spec/lib/lorekeeper/multi_logger_spec.rb

```diff
@@ -15,22 +15,22 @@
 
   context 'loggers added' do
     let(:message) { 'My heart aches, and a drowsy numbness pains my sense' }
+    let(:console_logger) { Lorekeeper::SimpleLogger.new(io) }
+    let(:json_logger) { Lorekeeper::JSONLogger.new(json_io) }
 
-    before do
-      logger.add_logger(io)
-      logger.add_logger(io2)
-    end
+    it 'calls all the methods of the loggers' do
+      logger.add_logger(console_logger)
+      logger.add_logger(json_logger)
 
-    it 'calls the methods of the loggers' do
-      logger.write(message)
-      expect(io.received_message).to eq(message)
-      expect(io2.received_message).to eq(message)
-    end
+      Lorekeeper::FastLogger::LOGGING_METHODS.each do |log_level|
+        logger.send(log_level, message) if logger.respond_to? (log_level)
 
-    it 'does not call the methods if they do not exist' do
```

Comment:
I refactored the initial loggers added spec to be able to test the changes.
Maybe I misunderstand something, but I don’t think we were testing correctly before 🤔 

When we’re adding loggers to Multilogger, we don’t add the io (e.g. STDOUT) itself, instead we initialize either json or simple logger and pass an io as an argument. Only then we pass the instance of simple or json loggers to the Multilogger.

Tests are passing after the refactoring and I hope they make sense.

Reply:
I think this was to ensure that if you do `logger.hello_world` somehow you did not end up with a log message. if we are removing method_missing I think this test is meaningless